source/meta.tex

```diff
@@ -6881,6 +6882,41 @@
 consteval info @\libglobal{unwrap_ref_decay}@(info type);
 \end{codeblock}
 
+\pnum
+For a function or function template $F$ defined in this subclause,
+let $C$ be its associated class template.
+For the evaluation of a call to $F$,
+let $S$ be the specialization of $C$ in terms of which the call is specified.
+\begin{itemize}
+\item
+  If
+  \begin{itemize}
+  \item
+    the template arguments of $S$ violate a condition specified
+    in a \Fundescx{Mandates} element in the specification of $C$;
+  \item
+    the call is specified to produce a reflection of a type,
+    but $S$ would have no member named \tcode{type}; or
+  \item
+    the call is specified to return \tcode{$S$::value},
+    but that expression is not a valid converted constant expression of type \tcode{R},
```

Comment:
The paper uses the indicative here, but on re-reading I think I meant to use the subjunctive just like the previous item, to avoid suggesting that `S::value` needs to be evaluated:

```suggestion
    but that expression would not be a valid converted constant expression of type \tcode{R},
```